### ISSUE TITLE
DSEV and MOLE are causing errors, corrupting savegames

### ISSUE THREAD

**Issue by Jimbodiah** [Angel-125/DSEV#9]
I had this with MOLE initially, but after removing that and trying DSEV I get the same issues described below, causing me to think it is related to the WButils perhaps.

After switching to another vessel the game will no longer save, return to KSC or switch to other vessels. When restarting KSP and going back to a ship, the same issues occurs again. The log shows the error "NullReferenceException: Object reference not set to an instance of an object"
 
[MOLE_KSP.zip](https://github.com/Angel-125/DSEV/files/1967945/MOLE_KSP.zip)

I reported this in your MOLE topic but never got a reply.

**Comment by Angel-125** [comment]
Need more info. Were you in flight?

**Comment by Jimbodiah** [comment]
Yes. It happens when I switch to a craft from the tracking station. I can play for a long time and no problem, then I switch to a ship with you parts and the error will come up.

**Comment by Angel-125** [comment]
Do you have experiments running? I've not seen this issue on my end.

**Comment by Jimbodiah** [comment]
No. I thought it was caused by MOLE first as it started occuring immediately after I sent up the bigby lab. But later when I installed just DSEV and started a new career, I had the same issues again. I am running 1.3.1 and DSEV 3.0.0.4 by the way (MOLE 1.10.2 at the time but have since uninstalled both mods).

The funky thing is I have used DSEV for ages and had no issues, I might try an older version to see if that helps.
